package.json

```diff
@@ -24,7 +24,7 @@
     "router"
   ],
   "scripts": {
-    "test": "jest --coverage --no-cache",
+    "test": "jest --coverage --no-cache --testURL https://hyperapp.js.org",
```

Comment:
Yeah, I agree it's a little weird. 

From what I understand, that might be the only way to instantiate a URL in the `jsdom` instances used by `jest`.  f you omit that `testURL`, it defaults to `about:blank`, which makes `window.location.origin === null`.  It throws `SecurityError` if you try to do a `history.pushState(...)` without setting a JSDOM url.